src/components/ScrollViewGesture.tsx

```diff
@@ -116,15 +116,18 @@ const IScrollViewGesture: React.FC<PropsWithChildren<Props>> = (props) => {
   );
 
   const endWithSpring = React.useCallback(
-    (onFinished?: () => void) => {
+    (scrollEndTranslationValue: number,
+      scrollEndVelocityValue: number,
+      onFinished?: () => void,
+    ) => {
       "worklet";
       const origin = translation.value;
-      const velocity = scrollEndVelocity.value;
+      const velocity = scrollEndVelocityValue;
```

Comment:
For now, I'm explicitly leaving both the function parameter `scrollEndVelocityValue` and this local const `velocity`.
We could (should?) eliminate this line, though, and instead rename the function parameter to `velocity` instead.